src/schema/search/SearchEntity.ts

```diff
@@ -48,6 +48,14 @@ export const SearchEntity = new GraphQLEnumType({
   },
 })
 
+const defaultBlockList = ["gallery", "institution"]
+
 export const DEFAULT_ENTITIES = SearchEntity.getValues().filter(
-  index => index.value !== "gallery" && index.value !== "institution"
+  index => !defaultBlockList.includes(index.value)
+)
+
+const suggestBlockList = ["gallery", "institution", "show"]
```

Comment:
My bad for not catching this, but this doesn't actually work. The values here come from the enum above, so its `PartnerShow`.